firebase-firestore/src/main/java/com/google/firebase/firestore/util/AsyncQueue.java

```diff
@@ -176,97 +176,180 @@ private void markDone() {
   }
 
   /**
-   * The single thread that will be used by the executor. This is created early and managed directly
-   * so that it's possible later to make assertions about executing on the correct thread.
+   * A wrapper around a {@link ScheduledThreadPoolExecutor} class that provides:
+   *
+   * <ul>
```

Comment:
Done.

Reply:
In javadoc this will now read:

```
1. 1. Synchronized ...
2. 2. Ability ...
3. 3. Single ...
```
`<ol>` makes each `<li>` insert a number instead of a bullet.